Decred Wallet/view_controller/ReceiveViewController.swift

```diff
@@ -45,17 +48,15 @@ class ReceiveViewController: UIViewController,UIDocumentInteractionControllerDel
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        self.subheader.text = LocalizedStrings.receiveHeaderInfo
-        // TAP Gesture
-        self.setupExtraUI()
+        
         self.starttime = Int64(NSDate().timeIntervalSince1970)
         setupSyncInProgressLabelConstraints()
     }
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         self.navigationItem.title = LocalizedStrings.receive
-        self.navigationItem.leftBarButtonItem = UIBarButtonItem(image: UIImage(named: "left-arrow"),
```

Comment:
Please check if this `left-arrow` is still used anywhere else in the app and remove it if not.

Reply:
I have checked it, this left-arrow is still used in some files in the app